From: John Ferlito <johnf@inodes.org>
Date: Fri, 22 Feb 2013 15:55:27 +1100
Subject: Debianise install paths

---
 etc/nfsen-dist.conf |   24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/etc/nfsen-dist.conf b/etc/nfsen-dist.conf
index cf08470..7d78caa 100644
--- a/etc/nfsen-dist.conf
+++ b/etc/nfsen-dist.conf
@@ -18,7 +18,7 @@
 
 #
 # Required for default layout
-$BASEDIR = "/data/nfsen";
+$BASEDIR = "debian/nfsen/usr";
 
 #
 # Where to install the NfSen binaries
@@ -26,17 +26,17 @@ $BINDIR="${BASEDIR}/bin";
 
 #
 # Where to install the NfSen Perl modules
-$LIBEXECDIR="${BASEDIR}/libexec";
+$LIBEXECDIR="${BASEDIR}/share/perl5/nfsen";
 
 #
 # Where to install the config files
-$CONFDIR="${BASEDIR}/etc";
+$CONFDIR="debian/nfsen/etc/nfsen";
 
 #
 # NfSen html pages directory:
 # All php scripts will be installed here.
 # URL: Entry point for nfsen: http://<webserver>/nfsen/nfsen.php
-$HTMLDIR    = "/var/www/nfsen/";
+$HTMLDIR    = "${BASEDIR}/share/nfsen/www";
 
 #
 # Where to install the docs
@@ -44,7 +44,7 @@ $DOCDIR="${HTMLDIR}/doc";
 
 #
 # Var space for NfSen
-$VARDIR="${BASEDIR}/var";
+$VARDIR="debian/nfsen/var/lib/nfsen";
 
 # directory for all pid files
 # $PIDDIR="$VARDIR/run";
@@ -60,15 +60,15 @@ $VARDIR="${BASEDIR}/var";
 #
 # The Profiles stat directory, where all profile information
 # RRD DBs and png pictures of the profile are stored
-$PROFILESTATDIR="${BASEDIR}/profiles-stat";
+$PROFILESTATDIR="${VARDIR}/profiles-stat";
 
 #
 # The Profiles directory, where all netflow data is stored
-$PROFILEDATADIR="${BASEDIR}/profiles-data";
+$PROFILEDATADIR="${VARDIR}/profiles-data";
 
 #
 # Where go all the backend plugins
-$BACKEND_PLUGINDIR="${BASEDIR}/plugins";
+$BACKEND_PLUGINDIR="${LIBEXECDIR}/plugins";
 
 #
 # Where go all the frontend plugins
@@ -76,7 +76,7 @@ $FRONTEND_PLUGINDIR="${HTMLDIR}/plugins";
 
 #
 # nfdump tools path
-$PREFIX  = '/usr/local/bin';
+$PREFIX  = "${BASEDIR}/bin";
 
 #
 # nfsend communication socket
@@ -88,12 +88,12 @@ $PREFIX  = '/usr/local/bin';
 # This may be a different or the same uid than your web server.
 # Note: This user must be in group $WWWGROUP, otherwise nfcapd
 #       is not able to write data files!
-$USER    = "netflow";
+$USER    = "www-data";
 
 # user and group of the web server process
 # All netflow processing will be done with this user
-$WWWUSER  = "www";
-$WWWGROUP = "www";
+$WWWUSER  = "www-data";
+$WWWGROUP = "www-data";
 
 # Receive buffer size for nfcapd - see man page nfcapd(1)
 $BUFFLEN = 200000;
