From: John Ferlito <johnf@inodes.org>
Date: Fri, 22 Feb 2013 15:58:00 +1100
Subject: Empty sources list

---
 etc/nfsen-dist.conf |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/etc/nfsen-dist.conf b/etc/nfsen-dist.conf
index 1ee090b..1d4f76a 100644
--- a/etc/nfsen-dist.conf
+++ b/etc/nfsen-dist.conf
@@ -160,9 +160,9 @@ $PROFILERS = 6;
 # Ident strings must be 1 to 19 characters long only, containing characters [a-zA-Z0-9_].
 
 %sources = (
-    'upstream1'    => { 'port' => '9995', 'col' => '#0000ff', 'type' => 'netflow' },
-    'peer1'        => { 'port' => '9996', 'IP' => '172.16.17.18' },
-    'peer2'        => { 'port' => '9996', 'IP' => '172.16.17.19' },
+#    'upstream1'    => { 'port' => '9995', 'col' => '#0000ff', 'type' => 'netflow' },
+#    'peer1'        => { 'port' => '9996', 'IP' => '172.16.17.18' },
+#    'peer2'        => { 'port' => '9996', 'IP' => '172.16.17.19' },
 );
 
 #
